### PR TITLE
[WIP] Mosaiq helper addition

### PR DESCRIPTION
**Pull request by rembishj**

### CODE REVIEW

File: pymedphys/_mosaiq/helpers.py [hunk 1]
```diff
@@ -319,3 +319,124 @@ def get_recently_completed_qcls_across_sites(
     results = results.sort_values(by="actual_completed_time", ascending=False)
 
     return results
+
+def get_mosaiq(mrn):
+    host_name = 'prwinvds006.utmsa.local'
```
Comment (rembishj) [inline pymedphys/_mosaiq/helpers.py]: this needs to be removed

File: pymedphys/_mosaiq/helpers.py [hunk 2]
```diff
@@ -319,3 +319,124 @@ def get_recently_completed_qcls_across_sites(
...
+    with mosaiq_connect(host_name) as cursor:
```
Comment (SimonBiggs) [inline pymedphys/_mosaiq/helpers.py]: Here's an example of the testing suite doing its thing:

https://github.com/pymedphys/pymedphys/pull/541/checks?check_run_id=264817076#step:10:339

That's saying `mosaiq_connect` is undefined in this file, which caused that test to fail. You'll notice I don't actually import `mosaiq_connect` here, I let that be done by the user of the function and then pass a "cursor" object to the function instead:

https://github.com/pymedphys/pymedphys/blob/7ea86ff38bcb6c32deaee94a5be252ed3ce28f46/pymedphys/_mosaiq/helpers.py#L37

**Comment by SimonBiggs** [comment]
@rembishj if you run `git pull` within your local pymedphys directory you should then be able to run `poetry install` as well as complete the environment set up docs.

**Comment by SimonBiggs** [comment]
@rembishj did you want to catch up this week?

**Comment by rembishj** [comment]
Yes, let's do that. Could you do your Friday morning/my thursday afternoon?

**Comment by SimonBiggs** [comment]
Yup let's do that, let's do the earlier time, same time as the original
PyMedPhys meeting so that I'm not getting interrupted with calls. I'll make
a meeting in Outlook when I get into work.

Have you guys had a daylight savings shift yet?

**Comment by SimonBiggs** [comment]
This has now been merged into master.

**Comment by SimonBiggs** [comment]
@rembishj would you be able to make a pull request with your logo adding yourself to the following list:

image

Make the logo 200x200 pixels and add it here:

https://github.com/pymedphys/pymedphys/tree/master/docs/logos

Also, here is the section of the README to edit:

https://github.com/pymedphys/pymedphys/blob/master/README.rst#L110-L190

**Pull request merged**